.github/workflows/pytests.yml

```diff
@@ -6,13 +6,51 @@ on:
       - main
   pull_request:
 
+
 jobs:
   test:
+    name: Run tests and collect coverage
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@main
-      - name: Run tests
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Setup Gardenlinux
+        uses: gardenlinux/python-gardenlinux-lib/.github/actions/setup@main
```

Comment:
Please remove `@main` here as we would like to be able to develop changes to the setup workflow in branches as well. :)

Reply:
Got it, removed and switched to relative path